tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -5,118 +5,159 @@ summary: Learn about the log auditing feature for the TiDB Cloud console.
+| CreatePrometheusIntegration | Create Prometheus integration |
+| DeletePrometheusIntegration | Delete Prometheus integration |
+| CreateCluster | Create a cluster |
+| DeleteCluster | Delete a cluster |
+| PauseCluster | Pause a cluster |
+| ResumeCluster | Resume a cluster |
+| ScaleCluster | Scale a cluster |
+| DownloadTiDBClusterCA | Download TiDB cluster CA certificate |
+| OpenWebSQLConsole | Connect to a TiDB cluster through Web SQL |
+| SetRootPassword | Set the root password of a TiDB cluster |
+| UpdateIPAccessList | Update the IP access list of a TiDB cluster |
+| DeleteAccessList | Delete the IP access list of TiDB cluster |
+| SetAutoBackup | Set the automatic backup mechanism of a TiDB cluster |
+| DoManualBackup | Perform a manual backup of TiDB cluster |
+| DeleteBackupTask | Delete a backup task |
+| DeleteBackup | Delete a backup file |
+| RestoreFromBackup | Restore to a TiDB cluster based on the backup files |
+| RestoreFromTrash | Restore to a TiDB cluster based on the backup files in the trash |
+| ImportDataFromAWS | Import data from AWS |
+| ImportDataFromGCP | Import data from GCP |
+| CreateMigrationJob | Create a migration job |
+| SuspendMigrationJob | Suspend a migration job |
+| ResumeMigrationJob | Resume a migration job |
+| DeleteMigrationJob | Delete a migration job |
+| ShowDiagnose | Show diagnosis information |
+| DBAuditLogAction | Set the activity of database audit logging |
+| AddDBAuditFilter | Add a database audit log filter |
+| DeleteDBAuditFilter | Delete a database audit log filter |
+| EditProject | Edit the information of a project |
+| DeleteProject | Delete a project |
+| BindSupportPlan | Bind a support plan |
+| CancelSupportPlan | Cancel a support plan |
+| UpdateOrganizationName | Update the organization name |
+| CreatePrivateEndpointService | Create private endpoint service |
+| DeletePrivateEndpointService | Delete private endpoint service |
+
+## Console audit log fields
+
+To help you track user activities, TiDB Cloud provides the following fields for each console audit log:
+
+| Field name | Data type | Description |
+|---|---|---|
+| type | string | Event type |
+| ends_at | timestamp | Event time |
+| operator_type | enum | Operator type: `user` or `api_key` |
+| operator_id | uint64 | Operator ID |
+| operator_name | string | Operator name |
+| operator_ip | string | Operator's IP address |
+| operator_login_method | enum | Operator's login method: `google`, `github`, `email`, or `api_key` |
+| org_id | uint64 | Organization ID to which the event belongs |
+| org_name | string | Organization name to which the event belongs |
+| project_id | uint64 | Project ID to which the event belongs |
+| project_name | string | Project name to which the event belongs |
+| cluster_id | uint64 | Cluster ID to which the event belongs |
+| cluster_name | string | Cluster name to which the event belongs |
+| trace_id | string | Trace ID of the request initiated by the operator |
```

Comment:
Other `x_id` fields are uint64, only trace_id is string. Please confirm this is not a mistake.